rls/src/actions/diagnostics.rs

```diff
@@ -331,7 +331,7 @@ mod diagnostic_message_test {
 
     impl FileDiagnosticTestExt for ParsedDiagnostics {
         fn single_file_results(&self) -> &Vec<(Diagnostic, Vec<Suggestion>)> {
-            self.diagnostics.values().nth(0).unwrap()
+            self.diagnostics.values().next().unwrap()
```

Comment:
This is nitpicking, but I find the `nth(0)` version more human-readable - otherwise, you need to decipher that we return an iterator and call `next`, which in this case would yield us first element. Could we please keep the old version?

Reply:
I've reverted that